doc/source/conf.py

```diff
@@ -356,4 +361,23 @@
     if (os.path.isfile(f) and f.endswith(
             ".rst") and f != "index.rst" and f != "modules.rst"):
         os.remove(f)
-apidoc.main([None, '-o', ".", "../../spinn_utilities"])
+
+
+# UGH!
+output_dir = os.path.abspath(".")
+os.chdir("../..")
+
+options = ['-o', output_dir, "spinn_utilities",
```

Comment:
Would it not be better to go down the arguments.extend(filtered_files(root_package, explicit_wanted_files)) route as done in PACMAN?

Reply:
Techniques are unified now